### ISSUE TITLE
365 learning pathways "custom learning admin" page blank  

### ISSUE THREAD

**Issue by Gmartin10** [pnp/custom-learning-office-365#450]
I've just provisioned a brand new pathways site and the "custom learning admin" section doesn't load on any browser' it's just a blank screen. The app catalog we have is fine

Can someone please advise a fix for this as I have raised with MS directly but they haven't been able to resolve.

**Comment by Gmartin10** [comment]
Please let me know any further info you need and i'll happily provide it.

**Comment by juliemturner** [comment]
Based on the error message in the console in the screen shot I think you either provisioned to a different site collection or changed the url of the site collection. Assuming that this is the url of the site collection you want registered as your main learning pathways site then you need to run a powershell script to update the tenant app property to indicate what the site collection url is of the root learning pathways site.

Here's a link to the script that updates the url: https://github.com/pnp/custom-learning-office-365/blob/main/installation/UpdateM365lpSiteUrl.ps1

**Comment by Gmartin10** [comment]
Hi Julie,

I've deleted the custom learning app from the App catalogue and re-provisioned the site several times. I have also deleted all other pathway sites we have had in existence and we still get this error.

Can I ask, Is it possible to have more than 1 pathways site URL in existence or does this cause issues?

![image](https://user-images.githubusercontent.com/78611922/107211008-dfa8f880-69fc-11eb-9c8e-a908d2d71edf.png)
